src/Utilities/FlowAnalysis/FlowAnalysis/Analysis/TaintedDataAnalysis/HardcodedSymmetricAlgorithmKeysSources.cs

```diff
@@ -0,0 +1,144 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using System.Linq;
+using Analyzer.Utilities.PooledObjects;
+using Microsoft.CodeAnalysis;
+
+namespace Analyzer.Utilities.FlowAnalysis.Analysis.TaintedDataAnalysis
+{
+    internal static class HardcodedSymmetricAlgorithmKeysSources
+    {
+        /// <summary>
+        /// <see cref="SourceInfo"/>s for hardcoded symmetric algorithm cryptographic keys tainted data sources.
+        /// </summary>
+        public static ImmutableHashSet<SourceInfo> SourceInfos { get; }
+
+        /// <summary>
+        /// Statically constructs.
+        /// </summary>
+        static HardcodedSymmetricAlgorithmKeysSources()
+        {
+            var builder = PooledHashSet<SourceInfo>.GetInstance();
+
+            builder.AddSourceInfo(
+                WellKnownTypeNames.SystemConvert,
+                isInterface: false,
+                taintedProperties: null,
+                taintedMethodsNeedsPointsToAnalysis: null,
+                taintedMethodsNeedsValueContentAnalysis: new (MethodMatcher, ValueContentCheck[])[]{
+                    (
+                        (methodName, arguments) =>
+                            methodName == "FromBase64String",
```

Comment:
```suggestion
                        (methodName, arguments) => methodName == "FromBase64String",
```

Reply:
You could also optionally discard the unused parameter: 
```suggestion
                        (methodName, _) =>
                            methodName == "FromBase64String",
```